ci/scripts/github/build.sh

```diff
@@ -30,11 +30,14 @@ sccache --version
 
 rapids-logger "Configuring cmake for Morpheus"
 git submodule update --init --recursive
+
+# Explicitly setting nvcc path Work-around for https://github.com/rapidsai/cudf/issues/12862
 cmake -B build -G Ninja ${CMAKE_BUILD_ALL_FEATURES} \
     -DCCACHE_PROGRAM_PATH=$(which sccache) \
     -DMORPHEUS_PYTHON_BUILD_WHEEL=ON \
     -DMORPHEUS_PYTHON_BUILD_STUBS=OFF \
-    -DCMAKE_BUILD_RPATH_USE_ORIGIN=ON .
+    -DCMAKE_BUILD_RPATH_USE_ORIGIN=ON \
+    -DCMAKE_CUDA_COMPILER=${CUDA_PATH}/bin/nvcc  .
```

Comment:
See PR #764

Reply:
I ran into a ton of trouble on this one, some of which is documented in https://github.com/rapidsai/cudf/issues/12862 

The issue is without specifying `CMAKE_CUDA_COMPILER` cmake will pick up `nvcc` from the conda env, which should be OK, but the problem is something in the stack (I'm not sure exactly what) is using the location of `nvcc` to infer the location of the CUDA libs, causing `libmorpheus.so` to link against the sub libs in the conda env. 

Without this calling `pybind11::module_::import("cudf")` or `CudfHelper::load()` causes a Runtime error because `cuInit` is undefined. However this only happens when Python is being used in embedded mode (ie. the C++ tests).